web/src/inbox_util.js

```diff
@@ -0,0 +1,59 @@
+import $ from "jquery";
+
+import * as compose_state from "./compose_state";
+import * as overlays from "./overlays";
+import * as popovers from "./popovers";
+import * as stream_color from "./stream_color";
+import * as stream_data from "./stream_data";
+
+let is_inbox_visible = false;
+
+export function set_visible(value) {
+    is_inbox_visible = value;
+}
+
+export function is_visible() {
+    return is_inbox_visible;
+}
+
+export function get_pm_key(msg) {
+    return "pm:" + msg.other_user_id;
+}
+
+export function is_in_focus() {
+    // Check if user is focused on
+    // inbox
+    return (
+        is_visible() &&
+        !compose_state.composing() &&
+        !popovers.any_active() &&
+        !overlays.is_overlay_or_modal_open() &&
+        !$(".home-page-input").is(":focus")
+    );
+}
+
+export function correct_stream_color(color) {
+    return color;
+}
+
+export function get_stream_header_color(color) {
+    return color;
+}
```

Comment:
I assume these two functions exist because there's design proposals that make these not the identify function? Probably would be worth some comments.

Reply:
Removed these, they were not being used anywhere.